cmds/core/dd/dd.go

```diff
@@ -51,17 +51,21 @@ import (
 	"github.com/u-root/u-root/pkg/progress"
 )
 
+type command struct {
+	skip    int64
+	seek    int64
+	conv    string
+	count   int64
+	inName  string
+	outName string
+	oFlag   string
+	status  string
+
+	bytesWritten int64 // access atomically, must be global for correct alignedness
+}
+
 var (
 	ibs, obs, bs *unit.Value
```

Comment:
staticcheck reports that ibs, ops, bs are unused as well, look like they are redeclared in main function

```go
	var (
		ibs = unit.MustNewUnit(ddUnits).MustNewValue(512, unit.None)
		obs = unit.MustNewUnit(ddUnits).MustNewValue(512, unit.None)
		bs  = unit.MustNewUnit(ddUnits).MustNewValue(512, unit.None)
	)
```

Reply:
removed. thanks.